app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+import app.errors as error
```

Comment:
It's better to import classes from module according to example in the checklist

![image](https://user-images.githubusercontent.com/104218550/219454942-68efce93-fb3d-4ce3-aee1-92d16c1c27ef.png)